contributed/plots/README copy.md

```diff
@@ -0,0 +1,5 @@
+# Plot Descriptions
```

Comment:
A comment on the filename: can you just call this README.md (not `README copy.md`)  so that when it's merged to master, and people navigate to contributed/plots/ in the DC2-analysis repo, they will see the readme in markdown format describing the plots?